csharp/Platform.Setters/Setter[TResult, TDecision].cs

```diff
@@ -30,29 +48,29 @@ public Setter(TDecision trueValue, TDecision falseValue, TResult defaultValue)
         }
 
         /// <summary>
-        /// <para>Gets result value.</para>
-        /// <para>Возвращает результирующее значение.</para>
+        /// <para>Initializes a new instance of the <see cref="Setter{TResult, TDecision}"/> class.</para>
+        /// <para>Иницаилизирует новый экземпляр класса <see cref="Setter{TResult, TDecision}"/>.</para>
```

Comment:
Arguments should be mentioned here.
You should add `param` tags for each argument too.